src/Platform.Handlers/tests/Xamarin.Platform.Handlers.DeviceTests/Stubs/Slider.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xamarin.Platform.Handlers;
+using Xamarin.Platform;
+using Xamarin.Forms;
+
+namespace Xamarin.Platform.Handlers.DeviceTests.Stubs
+{
+	public partial class SliderStub : StubBase, ISlider
```

Comment:
iOS doesn't support any sort of mocking library because of the limitations with Reflection.Emit so we have to do this all manually